rootfs/scheduler/__init__.py

```diff
@@ -1053,25 +1047,39 @@ def get_secrets(self, namespace, **kwargs):
 
         return response
 
-    def create_secret(self, namespace, name, data, secret_type='Opaque', labels={}):
+    def _build_secret_manifest(self, namespace, name, data, secret_type='Opaque', labels=None):
```

Comment:
It just seems unnecessary that we now have to determine the if `labels` is a `dict` type a little further down, whereas before we didn't have to because it was guaranteed to be a dictionary.